### PR TITLE
Changes to correct requirement for sentry - #177 

### PR DESCRIPTION
**Pull request by daveoconnor**
Also updated docs for the new local stack.
Seems like `.envrc` and `frontend/.env` / `frontend/src/.env.example` and documentation mentioning those could be removed.

### CODE REVIEW

**Comment by daveoconnor** [comment]
I'll test this again once the secrets PR is merged into main

**Comment by mgogoulos** [comment]
Haven't tried the changes, but reviewed the code and looks valid

**Comment by daveoconnor** [comment]
I've now changed this to handle the sentry changes, fixed the secrets issue in postgres yaml, and changed django.configmap.yaml to app.configmap.yaml. Ran a test project creation and everything spun up fine.

**Review by sjlawson — APPROVED** [review]

File: {{cookiecutter.project_slug}}/.envrc [hunk 1]
```diff
@@ -8,7 +8,3 @@ export POSTGRES_USER=$(op read "op://{{cookiecutter.project_name}}/POSTGERS_DB/u
 export POSTGRES_HOST=db-rw
```
[the remark this answers is not in the document]
Comment (daveoconnor) [reply {{cookiecutter.project_slug}}/.envrc]: Thanks, updated that and fixed another reference against a typod source in 1password that I saw.

**Pull request merged**